docs/setuptools.txt

```diff
@@ -1339,6 +1339,9 @@ Creating System Packages
     ``bdist_deb`` command is an example of a command that currently requires
     this kind of patching to work with setuptools.
 
+    Please note that building system packages may require you to install
+    some system software, e.g. ``bdist_rpm`` requires the ``rpmbuild``
```

Comment:
```suggestion
    some system software, for example ``bdist_rpm`` requires the ``rpmbuild``
```

Reply:
Not sure about the suggestion, AFAIR from school "for example" is never written but I'm not mother tongue in english :)

Reply:
"for example" is definitely written, "e.g." is used more in asides and is pronounced "ee gee". "For example" is more frequently used when the example is part of the flow of the sentence.